config/jobs/kubernetes/sig-release/release-branch-jobs/1.16.yaml

```diff
@@ -208,6 +208,60 @@ periodics:
   - 'perfDashPrefix: gce-100Nodes-1.16'
   - 'perfDashJobType: performance'
   - 'perfDashBuildsCount: 500'
+- annotations:
+    fork-per-release-cron: 0 15 * * *, 0 21 * * *
```

Comment:
Should be removed?

I guess I don't understand syntax of this annotation - it's different for every relkease.

Reply:
ditto